src/couch_dist/src/couch_dist.erl

```diff
@@ -33,6 +34,15 @@ childspecs() ->
         ]}
     ]}.
 
+listen(Name) ->
+    NodeName =
+        case is_atom(Name) of
+            true -> atom_to_list(Name);
+            false -> Name
+        end,
+    Mod = inet_dist(NodeName ++ "@127.0.0.1"),
```

Comment:
is it right to hardcode to 127.0.0.1 for everyone?

Reply:
Good point. I went by the fact that we have the `clouseau@127.0.0.1`. The right thing to do may be to see how `Host` is fetched for listen/2 and do the same thing. I'll push a fix

Reply:
It turned out a bit more involved than just grabbing `node()` and splitting it. It turns out `node()` doesn't return the node name yet as the distribution hasn't started. We can't query most of the `net_kernel` functions either, as we are in a callback, which `net_kernel` is calling, so we end up deadlocking. The only viable way to get host seemed to be perform the same parsing as out of init args as erl_distribution / net_kernel does.